src/evaluators/utility/PHAL_UnivariateDistribution.hpp

```diff
@@ -0,0 +1,300 @@
+//*****************************************************************//
+//    Albany 3.0:  Copyright 2016 Sandia Corporation               //
+//    This Software is released under the BSD license detailed     //
+//    in the file "license.txt" in the top-level Albany directory  //
+//*****************************************************************//
+
+#ifndef ALBANY_UNIVARIATEDISTRIBUTION_HPP
```

Comment:
Would you mind separating the words with an underscore? It makes the ifdef more readable. Also, throughout albany,